### PR TITLE
Adds crates to in-hand crafting. Satchels and Gear Harnesses too.

### PR DESCRIPTION
**Pull request by Owai-Seek**
## About The Pull Request

Adds a submenu for plasteel crafting to make most unsecured crate types. Adds Gear Harness and Satchels as craftables as well. Also changes crates to give plasteel. It's -technically- possible for crates to be profitable to craft, but its very little money.

## Why It's Good For The Game

I like organizing. Themed crates help me remain organized.

dreamseeker_8jQPnicFAH
ZeDAeC0ASu

## Changelog
:cl:
balance: Crates drop 2 plasteel instead of 2 iron now.
add: Crates, satchels, and gear harnesses can now be in-hand crafted with plasteel, cloth, and leather.
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -168,8 +168,24 @@ GLOBAL_LIST_INIT(plasteel_recipes, list ( \
...
+        new/datum/stack_recipe("freezer crate", /obj/structure/closet/crate/freezer, 2, time = 15, one_per_turf = 1, on_floor = 1), \
+        new/datum/stack_recipe("internals crate", /obj/structure/closet/crate/internals, 2, time = 15, one_per_turf = 1, on_floor = 1), \
+        new/datum/stack_recipe("hydroponics crate", /obj/structure/closet/crate/hydroponics, 2, time = 15, one_per_turf = 1, on_floor = 1), \
+        new/datum/stack_recipe("radiation crate", /obj/structure/closet/crate/radiation, 2, time = 15, one_per_turf = 1, on_floor = 1), \
+        new/datum/stack_recipe("engineering crate", /obj/structure/closet/crate/engineering, 2, time = 15, one_per_turf = 1, on_floor = 1), \
+        new/datum/stack_recipe("science crate", /obj/structure/closet/crate/science, 2, time = 15, one_per_turf = 1, on_floor = 1), \
+		new/datum/stack_recipe("trash cart", /obj/structure/closet/crate/trashcart, 2, time = 15, one_per_turf = 1, on_floor = 1), \
+		new/datum/stack_recipe("laundry cart", /obj/structure/closet/crate/trashcart/laundry, 2, time = 15, one_per_turf = 1, on_floor = 1), \
+	)), \
```
Comment (atakiya) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]:
```suggestion
	new /datum/stack_recipe_list("crates & carts", list( 
		new/datum/stack_recipe("grey crate", /obj/structure/closet/crate, 2, time = 15, one_per_turf = 1, on_floor = 1),
		new/datum/stack_recipe("medical crate", /obj/structure/closet/crate/medical, 2, time = 15, one_per_turf = 1, on_floor = 1),
		new/datum/stack_recipe("freezer crate", /obj/structure/closet/crate/freezer, 2, time = 15, one_per_turf = 1, on_floor = 1),
		new/datum/stack_recipe("internals crate", /obj/structure/closet/crate/internals, 2, time = 15, one_per_turf = 1, on_floor = 1),
		new/datum/stack_recipe("hydroponics crate", /obj/structure/closet/crate/hydroponics, 2, time = 15, one_per_turf = 1, on_floor = 1),
		new/datum/stack_recipe("radiation crate", /obj/structure/closet/crate/radiation, 2, time = 15, one_per_turf = 1, on_floor = 1),
		new/datum/stack_recipe("engineering crate", /obj/structure/closet/crate/engineering, 2, time = 15, one_per_turf = 1, on_floor = 1),
		new/datum/stack_recipe("science crate", /obj/structure/closet/crate/science, 2, time = 15, one_per_turf = 1, on_floor = 1),
		new/datum/stack_recipe("trash cart", /obj/structure/closet/crate/trashcart, 2, time = 15, one_per_turf = 1, on_floor = 1),
		new/datum/stack_recipe("laundry cart", /obj/structure/closet/crate/trashcart/laundry, 2, time = 15, one_per_turf = 1, on_floor = 1),
	)),
```

**Review by atakiya — CHANGES_REQUESTED** [review]
Why again are you changing it from iron to plasteel?

**Comment by Owai-Seek** [comment]
> Why again are you changing it from iron to plasteel?

Crates sell for 200 credits in cargo, so if you slice open a crate for the plasteel, you get 150 credits back (or, well, the plasteel to re-make the crate)

If you could make the crates out of iron, a lot of stuff would have to be re-addressed in the cargo code, because everything in cargo is based off of cargo crate values.

**Review by atakiya — APPROVED** [review]
As discussed in Discord, not happy with the change to plasteel, but its due to other issues regarding cargo economics.

Should be fine for now in this case.

**Pull request merged**